x-pack/test_serverless/api_integration/test_suites/observability/cases/get_case.ts

```diff
@@ -36,7 +36,6 @@ export default ({ getService }: FtrProviderContext): void => {
       const theCase = await svlCases.api.getCase(
         {
           caseId: postedCase.id,
-          includeComments: true,
```

Comment:
Could you please add a test that does this but expects 400? (for security also)

Reply:
Good point. Thanks for that!

Reply:
Done in [`b2fecbd` (#198378)](https://github.com/elastic/kibana/pull/198378/commits/b2fecbd875a88b42abde434ff13727e7e029f456)